README.md

```diff
@@ -1,49 +1,50 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# React test workshop
 
-## Backend
-
-In order to temporarily store the todo list we're using a very simple express server.
-The backend can be started by running the command `node ./server.js`.
+Nå ska vi lære oss at teste React-kode!
 
-## Available Scripts
+## Kom i gang
 
-In the project directory, you can run:
+Det finnes en tilhørende [presentasjon](https://joakimgy.github.io/react-test-workshop/#/) som kan være til hjelp med at komme igang med denne workshop. Ellers er det bare å følge trinnene nedenfor for å komme i gang!
 
-### `yarn start`
+1. Last ned repoet ved å kjøre `git clone https://github.com/joakimgy/react-test-workshop.git` i terminalen.
+2. Navigere til root-folderen i terminalen.
```

Comment:
Her bør dere legge til et notat om å kjøre npm install, og kanskje tipse folk om hvordan de kan sette opp npm og node om de ikke har det installert

Reply:
Godt poeng der med hvordan sett opp npm og node! Skal se på det 👍

Reply:
Kan legge til en `Forutsetninger / Det må du ha før du starter` i `Kom i gang`. 
Denne lenken ser bra ut for install av `node` og `npm` på Mac https://treehouse.github.io/installation-guides/mac/node-mac.html 
Litt mer usikker om denne for Windows: https://phoenixnap.com/kb/install-node-js-npm-on-windows 
Evt Linux Ubuntu: https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-18-04 (latest LTS)

Reply:
Super! La til dem alle :)